dashboard/app/models/levels/blockly.rb

```diff
@@ -301,7 +301,7 @@ def localized_blockly_level_options(script)
 
         set_unless_nil(level_options, 'longInstructions', localized_long_instructions)
         set_unless_nil(level_options, 'failureMessageOverride', localized_failure_message_override)
-        set_unless_nil(level_options, 'startHtml', localized_start_html(level_options['startHtml']))
+        # set_unless_nil(level_options, 'startHtml', localized_start_html(level_options['startHtml']))
```

Comment:
If we aren't reverting all work and instead doing this and commenting out its use, I would add a comment to justify the commented out line. Something to the effect: "We are not localizing start_html content until we address issues that break current curriculum for students"

This is just in case anybody comes across this before it is actually addressed (or we somehow don't address it.)

You could even annotate with a Jira ticket, if possible, that tracks the work to eventually turn this back on. "See Jira P20-XX"

Reply:
I see it would be P20-102.

Reply:
Done!!